tests/test_rows.py

```diff
@@ -56,6 +56,12 @@ def test_namedtuple_row(conn):
     assert r2.number == 1
     assert not cur.nextset()
     assert type(r1) is not type(r2)
+    
```

Comment:
Also, this line has trailing whitespaces:
```suggestion

```

Reply:
Thanks. I will install black before next PR.